plugins/modules/delete_service_account_by_id.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Apache License, v2.0 (https://www.apache.org/licenses/LICENSE-2.0)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: delete_service_account_by_id
+
+short_description: Delete a Service Account for use with Red Hat Openshift Application Services 
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Delete a Service Account for use with Red Hat Openshift Application Services 
+
+options:
+    service_account_id:
+        description: ID of the Service Account
+        required: true
+        type: str
+ 
+# Specify this value according to your collection
+# in format of namespace.collection.doc_fragment_name
+extends_documentation_fragment:
+    - redhat.rhoask.my_doc_fragment_name
+
+author:
+    - Red Hat Developer
+'''
+
+EXAMPLES = r'''
+# Pass in a message
+  - name: Delete Service Account
+    delete_service_account_by_id:
+        service_account_id: "service_account_id"
+'''
+
+RETURN = r'''
+# These are examples of possible return values, and in general should use other names for return values.
+original_message:
+    description: The original params that were passed in to be deleted.
+    type: dict 
+    returned: always in case of success
+message:
+    description: The output error / exception message that is returned in the case the module generates an error / exception.
+    type: dict
+    returned: in case of error / exception
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+import rhoas_service_accounts_mgmt_sdk
+import auth.rhoas_auth as auth
+from rhoas_service_accounts_mgmt_sdk.api import service_accounts_api
+
+def run_module():
+    # define available arguments/parameters a user can pass to the module
+    module_args = dict(
+        service_account_id=dict(type='str', required=True),
+    )
+    
+    result = dict(
+        changed=False,
+        original_message='',
+        message='',
+    )
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    if module.check_mode:
+        module.exit_json(**result)
+
+    token = auth.get_access_token()
+    
+    configuration = rhoas_service_accounts_mgmt_sdk.Configuration(
+        host = "https://sso.redhat.com/auth/realms/redhat-external",
```

Comment:
Antipattern. We should not have it hardcoded etc.

Reply:
No. Having constant used across modules is good

Reply:
but thats not really possible as there are two host addresses, we've got that `sso...` for service accounts and `api.openshift...` for kafka mgmt 🤔

Reply:
Not possible to have 2 constants?

Reply:
I'm looking to have central point for all configuration based elements. In future we can seed them from rhoas/config.json for now just 2 constants in one file